### ISSUE TITLE
R50 panoptic eval 

### ISSUE THREAD

**Issue by nazimii** [IDEA-Research/MaskDINO#23]
hi,I found there are some problems when eval R50 panopric（53.0）when load the model（download from your github）
<img width="896" alt="image" src="https://user-images.githubusercontent.com/41355745/207043356-923ea22d-f753-4815-be48-e2e2472ca3ad.png">

<img width="847" alt="image" src="https://user-images.githubusercontent.com/41355745/207043436-3763cb0c-d210-4c23-9fce-e8fac35f425a.png">

so I think this mistake is the mismatch between checkpoint and model

please check it!! Very thanks!!!

**Comment by nazimii** [comment]
or this parameters are not used when panoptic ？

**Comment by nazimii** [comment]
ok，thanks！

**Comment by nazimii** [comment]
hi, I have a new question
(1) are you provide  pretrained backbones weight？

<img width="696" alt="image" src="https://user-images.githubusercontent.com/41355745/207078101-b9a02763-a2f3-4372-babe-8de2b562aaee.png">

so that I can retrain model start from scratch, 
(2) which dataset you used to pretrain swinL? in paper you say object365,but in github it is IN21K

**Comment by FengLi-ust** [comment]
1. The weight is trained on the COCO dataset. You can download and evaluate it to reproduce our results.
2. We have not released the results pretrained on O365. The released results are purely trained on the corresponding dataset without any extra data or tricks.

**Comment by nazimii** [comment]
for panoptic：
（1）R50，you use R50 weight（imagenet pretrain），and train it on  COCO
（2）swinL，you use Swin-L weight（IN21K pretrain），and train it on 365O，and finally train it on COCO？
so 365O is just used at swin-L exam？

**Comment by nazimii** [comment]
ok~~,
I have a new question:
(1) your lr_base is 0.0001 when use 8GPU?, so I just have one GPU, it should be 0.0001/8 ?
(2) how many iter in one epoch? 36875?
<img width="321" alt="image" src="https://user-images.githubusercontent.com/41355745/207269063-5adfc448-689a-4ba6-ac11-f1b7b825bdba.png">
(3) you use 8GPU and set batchsize = 16 ,so there are two input img at each GPU?

**Comment by nazimii** [comment]
I try to retrain R50 Mask MINO, with your pretrain weight ,and set LR = 0.0001 ,bs = 2 for one GPU, get an worse result(around 45 PQ), I just retrain it for 30000step, but it should be a value near 53 PQ as your paper's result?

<img width="582" alt="image" src="https://user-images.githubusercontent.com/41355745/207271312-1bc1e707-502b-4b94-bc7f-db88b6cb31eb.png">

**Comment by nazimii** [comment]
can you provide your training log (R50 panoptic at COCO)，very thanks！

**Comment by FengLi-ust** [comment]
Your batchsize is not the same as ours. We use total batchsize 16. DETR-based models are sensitive to batchsize.

We did not try using other batchsize, maybe you need to explore it yourself.

**Comment by nazimii** [comment]
so if I run: python train_net.py --num-gpus 8 --config-file config_path
for R50 panoptic, I can get 53.0 after 50 epoch ? without your pretrain weight

**Comment by nazimii** [comment]
can you provide your training log at R50 COCO panoptic, so that I can have a reference

**Comment by FengLi-ust** [comment]
Yes, you are expected to get around 53.0 PQ after 50 epochs. Our original log is disordered. We will consider retraining the model and releasing the training log in the future.

**Comment by nazimii** [comment]
how many iter in one epoch? 368750?
in config max_iter is 368750

**Comment by HaoZhang534** [comment]
@nazimii There are 7375 iters in one epoch for coco.
